lib/pure/includes/osenv.nim

```diff
@@ -247,7 +134,22 @@ else:
       else:
         if c_unsetenv(key) != 0'i32:
           raiseOSError(osLastError())
-      environment.delete(indx)
+
+  when (defined(macosx) and not defined(ios) and not defined(emscripten)) or defined(nimscript):
+        # From the manual:
```

Comment:
indentation

Reply:
What's wrong with this indentation? (This whole block is in an else: after checking for js)

Reply:
style guide says 2 spaces for indentation

Reply:
That's what line 138 is?